src/components/home/Popup.tsx

```diff
@@ -0,0 +1,184 @@
+    position: 'fixed',
+    margin: 0,
+    padding: 0,
+    display: 'flex',
+    flexDirection: 'row',
+    backgroundColor: 'popup.popupBg',
+    top: 0,
+    left: 0,
+    zIndex: 1000,
+
+    '@keyframes fadeIn': fadeIn,
+    animationName: 'fadeIn',
+    animationDuration: '0.25s',
+  }
+
+  const photoRegionDiv: SxStyleProp = {
+    height: '100vh',
+    display: 'flex',
+    flexDirection: 'column',
+    width: ['100vw', '65vw'],
+    ml: 0,
+    mr: 'auto',
+  }
+
+  const textRegionDiv: SxStyleProp = {
+    height: '100vh',
+    width: '35vw',
+    mr: 0,
+    ml: 'auto',
+    px: '2%',
+    py: '2%',
+    backgroundColor: 'popup.textBg',
+    visibility: ['collapse', 'visible'],
+  }
+
+  const authorTextStyle: SxStyleProp = {
+    fontSize: 'medium',
+    fontFamily: 'body',
+  }
+
+  const lineStyle: SxStyleProp = {
+    border: '1px solid gray',
+    height: '2px',
+    width: '100%',
+    margin: 'auto',
+    mb: '7%'
+  }
+
+  const textBoxStyle: SxStyleProp = {
+    overflowY: 'auto',
+    width: '100%',    
+    maxHeight: '85%',
+    '&::-webkit-scrollbar': {
+      width: '0.5em'
+    },
+    '&::-webkit-scrollbar-thumb': {
+      backgroundColor: 'rgba(0, 0, 0, 0.05)',
+      outline: '1px solid slategrey'
+    }
+  }
+
+  const borderStyle: SxStyleProp = {
+    backgroundColor: 'primary',
+    width: '100%',
+    height: '7%',
+    display: 'flex',
+    flexDirection: 'row'
+  }
+
+  const photoContainerStyle: SxStyleProp = {
+    height: `${100 - parseInt(borderStyle.height.toString()) * 2}%`,
+    width: '100%',
+    textAlign: 'center',
+    verticalAlign: 'middle',
+    display: 'flex',
+    justifyContent: 'center'
+  }
+
+  const photoStyle: SxStyleProp = {
+    objectFit: 'contain',
+    maxHeight: '100%',
+    maxWidth: '100%',
+    height: 'auto',
+    width: 'auto',
+  }
+
+  const returnButtonStyle: SxStyleProp = {
+    my: 'auto',
+    ml: '1%',
+    color: 'white'
+  }
+
+  /**
+   * Consumes (stops the propagation of) a mouse event.
+   * 
+   * Nested DOM elements propagate events to parent DOM elements, which can
+   * occasionally be unideal. This method stops the propagation.
+   * 
+   * @param evt The mouse event to be consumed.
+   */
+  const consumeMouseInteraction = (evt: React.MouseEvent<HTMLElement>): void => {
+    evt.stopPropagation();
+  }
+
+  // In the event that this popup is created somehow with no photo, generate
+  // empty element instead of throwing an error.
+  if (!photo) {
+    return <React.Fragment></React.Fragment>
+  }
+
+  return (
+    <div sx={popupDiv}>
+      <div sx={photoRegionDiv} onClick={closeHandler}>
+        <div sx={borderStyle}>
+          <RiCloseCircleLine color='white' size={30} sx={returnButtonStyle} onClick={closeHandler}/>
+        </div>
+        <div sx={photoContainerStyle}> 
+          <img src={photo.url} alt={photo.name} sx={photoStyle} onClick={consumeMouseInteraction} />
+        </div>
+        <div sx={borderStyle} />
+      </div>
```

Comment:
Why don't you make the containing div (`photoRegionDiv`) background dark blue and have it close if you click anywhere that's not the photo itself. Right now, it doesn't close a landscape photo unless you close on the border (nothing happens when you click on the extra space).

Reply:
Writing an issue for this.